### PR TITLE
Bilinear Search in C

### PR DESCRIPTION
**Pull request by sparkingdark**
new file:   C/search/bilinear.c

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes i read it.

### Description
It is the implementation of bilinear search in c regarding issue #195 
(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#1270

### CODE REVIEW

File: C/search/bilinear.c [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    {
+        /* display the array */
+        printf("%d ",*(a+i));
+    }
+    
+}
+
+void bilinear_search(int *a,int ele,int n){
+
+    int i = 0;
```
Comment (atarax665) [inline C/search/bilinear.c:32]: Use meaningful name for the variables instead of i, j, p.

File: C/search/bilinear.c [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    int *a = create(n);
+
+    display(a,n);
+
+    printf("\n enter the element to search:");
+    scanf("%d",&ele);
+
+    bilinear_search(a,ele,n);
+    return 0;
+}
```
Comment (atarax665) [inline C/search/bilinear.c:85]: Add sample input/output and time/space complexity of the algorithm in commented form.

**Review by atarax665 — CHANGES_REQUESTED** [review]
Format your code according to C style guide.

**Comment by atarax665** [comment]
Update README.md

**Comment by sparkingdark** [comment]
Okay I will do the requested changes.

**Comment by HarshCasper** [comment]
I'm closing the PR due to no activity. If you would like to reopen this PR to be reviewed please join the [Slack](https://join.slack.com/t/tesseractcodi-bxq5968/shared_invite/zt-ju6c0nqq-KgkTvFyxjNVbDRBIjADanw) and drop me a message.

**Pull request closed without merging**